tests/wpt/mozilla/tests/mozilla/interfaces.js

```diff
@@ -34,15 +33,12 @@ function test_interfaces(interfaceNamesInGlobalScope) {
       "ReferenceError",
       "Reflect",
       "RegExp",
-      "SIMD",
       "Set",
       "SharedArrayBuffer",
-      "StopIteration",
       "String",
       "Symbol",
       "SyntaxError",
```

Comment:
Should SubtleCrypto be in this file too?

Reply:
I think ATM we're not implementing SubtleCrypto, when we are we should add it here.